crates/stc_ts_type_checker/tests/conformance/types/localTypes/localTypes4.ts.stats.rust-debug

```diff
@@ -1,6 +1,6 @@
 Stats {
     required_error: 2,
     matched_error: 2,
-    extra_error: 0,
+    extra_error: 1,
```

Comment:
This should be fixed

Reply:
```ts
function f4() {
    // Local types are block scoped
    interface T { x: number }
    let v: T;
    v.x = 10;
    if (true) {
        interface T { x: string }
        let v: T;
        v.x = "hello"; // extra_error
    }
    else {
        v.x = 20;
    }
}
```
For this to be fixed, the scope must be separated.

Reply:
If so, let's postpone this. Thank you!